.travis/gtest_dlt_all.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
the travis ci is deprecated and was removed. Please remove it from this commit as well.

Reply:
updated fix